examples/polynomial/evaluation/ensemble.yml

```diff
@@ -1,4 +1,4 @@
-size: 1000
+size: 1001
```

Comment:
Good 👏 Now we have at least two integration tests with different ensemble sizes!